webpack.config.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-env node */
+
+/* TODO:
```

Comment:
are you going to create issues for those TODOes ?

Reply:
Some of them are already in progress, others I'll create issues.